kube-burner-workload/kind-metrics/README.md

```diff
@@ -0,0 +1,20 @@
+kubectl apply -f monitoring.yaml
+
+docker run -p 9200:9200 -d -e "discovery.type=single-node" -e "plugins.security.disabled=true" docker.io/opensearchproject/opensearch:latest
+docker run -d --name=grafana -p 3000:3000 grafana/grafana:latest
+
+kube-burner init -m ./metrics.yml -c ../network-policy.yaml -u http://<prometheus pod node ip>:<prometheus service nodePort>
+
+All creds are admin/admin
+
+Grafana should be at localhost:3000
+1. configure Data Sources (Menu > Connections > Data sources)
+Add data source of type elasticsearch
+URL = http://admin:admin@<opensearch container ip>:9200
```

Comment:
how opensearch is related to elastic search?

Reply:
updated to use elasticsearch to avoid confusion :)